### PR TITLE
Changes to make udp port check for pre-deployment

### PR DESCRIPTION
**Pull request by vipin230**
### :nut_and_bolt: Description: What code changed, and why?
Changed udp port check to not trigger if deployment state is post-deployment

### :chains: Related Resources

https://chefio.atlassian.net/browse/CHEF-3884

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
Pre-deployment :-
Screenshot 2023-07-04 at 7 30 13 PM
Screenshot 2023-07-04 at 7 30 19 PM

Post-deployment :-
Screenshot 2023-07-04 at 7 51 51 PM
Screenshot 2023-07-04 at 7 51 18 PM

### CODE REVIEW

File: components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go [hunk 1]
```diff
@@ -142,13 +142,15 @@ func getRequestsForPostgresAsSource(config *models.Config) []models.FirewallRequ
 				continue
 			}
 			//for Upd
```
Comment (vivek-yadav) [inline components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go:144]:
```suggestion
			//for Udp
```

File: components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go [hunk 2]
```diff
@@ -142,13 +142,15 @@ func getRequestsForPostgresAsSource(config *models.Config) []models.FirewallRequ
...
-			reqBody := models.FirewallRequest{
-				SourceNodeIP:               sourceNodeIP,
-				DestinationNodeIP:          destNodeIP,
-				DestinationServicePort:     "9638",
-				DestinationServiceProtocol: "udp",
+			if config.DeploymentState != "post-deploy" {
```
Comment (vivek-yadav) [inline components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go:145]: make this "post-deploy" a const, then compare it here.
```suggestion
			if config.DeploymentState != POST_DEPLOY_STATE {
```

File: components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go (hunk 1 above)
Comment (vipin230) [reply components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go]: done

File: components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go (hunk 2 above)
Comment (vipin230) [reply components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/firewallchecktrigger/firewallchecktrigger.go]: done

**Review by vivek-yadav — APPROVED** [review]

**Pull request merged**